buildozer/targets/android.py

```diff
@@ -200,6 +200,20 @@ def _sdkmanager(self, *args, **kwargs):
         kwargs['cwd'] = kwargs.get('cwd', android_sdk_dir)
         command = [self.sdkmanager_path, f"--sdk_root={android_sdk_dir}", *args]
 
+        # Attempt to pass proxy settings to sdkmanager command using the --proxy, --proxy_host,
+        # and --proxy_port arguments by checking common environment variables for proxy settings.
+        # Sdkmanager only supports proxy types `http` and `socks`, so even if proxy url is https,
+        # only pass host and port with `http` type.
+        for key in ['HTTP_PROXY', 'http_proxy', 'HTTPS_PROXY', 'https_proxy']:
+            proxy = os.environ.get(key)
+            try:
+                host, port = proxy.split(':')[-2:]
```

Comment:
`proxy = os.environ.get(key)` could be set directly with `host, port = proxy.split(':')[-2:]` so it becomes `host, port = os.environ.get(key).split(':')[-2:]`